app/Image/GdHandler.php

```diff
@@ -53,7 +53,20 @@ public function scale(
 		}
 
 		imagecopyresampled($image, $sourceImg, 0, 0, 0, 0, $newWidth, $newHeight, $width, $height);
-		imagejpeg($image, $destination, $this->compressionQuality);
+
+		switch ($mime) {
+			case IMAGETYPE_JPEG:
+			case IMAGETYPE_JPEG2000:
+				imagejpeg($image, $destination, $this->compressionQuality);
+				break;
+			case IMAGETYPE_PNG:
+				imagepng($image, $destination);
```

Comment:
you are missing the compression here: 
https://www.php.net/manual/en/function.imagepng.php

Reply:
What value would you like me to provide though? According to the manual, _The default (-1) uses the zlib compression default_, so I figured I didn't need to provide anything. And the config we use for JPEGs is in a different range (0-100 vs 0-10 here).

Reply:
Well, divide my 10 and round. :) What's the problem ? :smile:

Reply:
Also as it's lossless, it has quite a different meaning.